extras.json

```diff
@@ -33,8 +33,11 @@
   "sky": {
     "strength": 6,
     "color": [0.854, 0.862, 1],
+    "ambientLight": {
```

Comment:
The cubes.zip example has to be updated.

Reply:
When you have new cubes.zip could you also update e2e_tests in the shapespark repo?

Reply:
I've updated cubes.zip in this repo and in the `shapespark` repo.